install-dev/init.php

```diff
@@ -78,7 +78,7 @@
 
 @set_time_limit(0);
 if (!@ini_get('date.timezone')) {
-    @date_default_timezone_set('UTC');
```

Comment:
because UTC does not exists, and Sf will check existence of timezone during install (blocking)